ide/che-core-ide-jseditor/src/main/java/org/eclipse/che/ide/jseditor/client/link/LinkedModel.java

```diff
@@ -14,6 +14,10 @@
 import java.util.List;
 
 /**
+ * The model for linked mode, umbrellas several
```

Comment:
This javadoc may be formatted in better way.

* Separate description from details.
* Separate <code>@author</code> from other stuff 

for instance:

```java
 /**
  * The model for linked mode, umbrellas several {@link LinkedModelGroup model groups}.
  * 
  * <p>Once installed ,the model propagates all changes 
  * to its siblings in the same position group.
  * 
  * @author Evgen Vidolob
  */
```